docker-compose.yml

```diff
@@ -12,10 +12,12 @@ mongodb:
   image: mongo:latest
   volumes_from:
     - mongodb-data
+  volumes:
+    - ./data/archive:/archive
```

Comment:
I'm not sure what the point of this is, as far as I know mongo does nothing with `/archive`, so your `./data/archive` is most likely empty?